.github/CODEOWNERS

```diff
@@ -928,6 +927,8 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+/sdk/cdn/                                                          @ptnan7
```

Comment:
Please follow the form of the other entries here.   Restrict this to the fully-qualified path for the resource manager package, add a PR label decorator to the path, add a service label + service owners block.